core/templates/services/context.service.spec.ts

```diff
@@ -160,6 +204,17 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('topic_editor');
```

Comment:
Where do we set the page context in this test? Also, I don't see a precheck for this test. Please add one.

Reply:
Ditto: spyOn(urlService, 'getPathname').and.returnValue('/topic_editor/123'); in beforeEach is cause of this

Reply:
Can we set the context in this test and have a precheck as well?

Reply:
Done!